### ISSUE TITLE
[Accessibility]A11y__AzureDataStudio_NewJobSteps_Keyboard: "Open " files button is not accessible by keyboard.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#5783]
**Bug Description:**
[Check out [Keros!]](https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx) Identify accessibility bugs before check-in and make bug fixing faster and easier.
 GitHubTags: #A11yMAS; #A11yTCS; #MAS2.1.1; #A11y_AzureDataStudio; #Win32; #DesktopApp
**Environment Details:**
Application name: Azure Data Studio
Version: 1.8.0-insider (user setup)
Commit: 3e8f25d
Date: 2019-05-27T07:46:13.508Z
Electron: 3.1.8
Node.js: 10.2.0
OS: Windows_NT x64 10.0.17763

**Additional details:**
Tool name: Keyboard
MAS Reference: MAS2.1.1

**Steps to Reproduce:**
1.Launch the Azure Data Studio application.
2.Enable the SQL server agent extension.
3.Navigate to the SQL agent tab from Home page->Click on new job button.
4.Fill in the details in the "Name" field->navigate to the steps tab->Click on "New Step" button.
5.Fill in all the required details in the pop-up and navigate to the "Open" button.
6.Try to activate the keyboard through the enter/spacebar key.
7.Observe of the pop-up to select the files opens up.
   
Actual:
"Open " files button is not accessible by keyboard.

**Expected:**
"Open " files button should be accessible by keyboard on pressing the enter/spacebar key.

**User Impact:**
Keyboard only users will not be open any of the files folder to the given path using keyboard strokes.

**Recommendation:**
Please refer the following snippet link for fixes:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx
**MAS Reference:**
**MAS2.1.1-**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

**Attachment for Reference**
[5783_BugAttcahment.pptx](https://github.com/microsoft/azuredatastudio/files/3241188/5783_BugAttcahment.pptx)

Does this issue occur when all extensions are disabled?: Yes
